docs/automated_testing.md

```diff
@@ -930,6 +991,36 @@ uses: suffolkLITLab/ALKiln@releases/v4
 
 When you want to update to a new version of the ALKiln, update that sha manually.
 
+### GitHub secrets
+
+Use GitHub secrets to set variable values with sensitive information. For example, a password. The value of this variable will not appear anywhere in the report or in the console. You also will be unable to take a screenshot of the page.
+
+1. Follow the [GitHub instructions](https://docs.github.com/en/actions/security-guides/encrypted-secrets) to set one or more GitHub secrets. You can add these to one repository secrets or you can add these to your organization secrets, whichever is right for you.
+
+2. Go to the home page of your repository. Tap on the `.github` folder, then on `workflows` folder, then on the YAML file in there that runs the ALKiln tests.
+
+It should include a line that looks like this:
+```yml
+      - uses: actions/checkout@v2
+```
+
+3. If this is the first environment variable you're adding, add these lines:
+
+```yml
+      - name: Set env vars
+        run: |
+```
+
+4. Whenever you want to add a secret, add a new line under those in this format:
+
+```yml
+          echo "USER_PASSWORD=${{ secrets.USER_PASSWORD }}" >> $GITHUB_ENV
```

Comment:
Are you sure it does that when using the composite action? That is, from a dependent's repo? In which case, where does it go?

I thought I tried that and it didn't work, but I could be wrong.

Reply:
It's directly above `job`. A live version is here: https://github.com/SuffolkLITLab/docassemble-EFSPIntegration/blob/main/.github/workflows/kiln_tests.yml.  It's failing, but I'm pretty sure that's because it needs https://github.com/SuffolkLITLab/ALKiln/pull/581, going into the logs you see where it gets past the steps that read in those environment vars.